crt_portal/static/js/complaint_view_filters.js

```diff
@@ -53,7 +53,7 @@
    */
   function makeQueryParams(params) {
     var keys = Object.keys(params);
-
+    console.log("Make query params")
```

Comment:
please remove debug statements like this.
Plus, the changes in this file are all spacing related, so don't need to check this in